deployment/ccip/changeset/testhelpers/test_environment.go

```diff
@@ -570,9 +591,11 @@ func NewEnvironmentWithJobsAndContracts(t *testing.T, tEnv TestEnvironment) Depl
 	evmChains := e.Env.AllChainSelectors()
 	solChains := e.Env.AllChainSelectorsSolana()
 	aptosChains := e.Env.AllChainSelectorsAptos()
+	tonChains := e.Env.AllChainSelectorsTon()
```

Comment:
You also need to update the ValidateSelectorsInEnvironment() in chainlink/deployment/helpers.go

Reply:
The next step would be implement DeployLinkToken() for TON

Reply:
> You also need to update the ValidateSelectorsInEnvironment() in chainlink/deployment/helpers.go

Yep, I had this change locally, just pushed it